plugins/dell-dock2/fu-dell-dock2-rtshub-firmware.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2024 Dell Technologies
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include "fu-dell-dock2-common.h"
+#include "fu-dell-dock2-rtshub-firmware.h"
+
+#define DOCK_RTSHUB_GEN2_VERSION_OFFSET 0x7F52
+#define DOCK_RTSHUB_GEN1_VERSION_OFFSET 0x7FA6
+#define DOCK_RTSHUB_GEN1_VID_OFFSET	0x7FA8
+#define DOCK_RTSHUB_GEN1_PID_OFFSET	0x7FAA
+
+#define DELL_VID 0x413C
```

Comment:
this is also in fu-dell-dock2-common.h, can be dropped here

Reply:
Fixed, thanks.